prow/cmd/tackle/main.go

```diff
@@ -98,6 +103,15 @@ func projects(max int) ([]string, error) {
 	return strings.Split(out, "\n"), nil
 }
 
+// projects returns the list of accessible gcp zones
```

Comment:
```suggestion
// zones returns the list of accessible gcp zones
```

Reply:
I was trying to stay aligned with the comment conventions, so did a bit of copy/paste here. I must have missed this. I will update!